python/README.rst

```diff
@@ -36,14 +53,13 @@ Install python3.7 and use it for this directory
 .. code-block:: bash
 
     cd looker-sdk-example/
-    pyenv install 3.7.4
     pyenv local 3.7.4
 
 Install looker_sdk using pipenv
 
 .. code-block:: bash
 
-    $ pipenv install --pre looker_sdk
+    pipenv install --python 3.7 --pre looker_sdk
```

Comment:
What do you think of splitting this into two commands? Clearer, unnecessary or more confusing? 
```
pipenv shell --python 3.7.4
pipenv install --pre looker_sdk
```

Reply:
I don't think there's a reason to launch them into `pipenv shell` I'd rather stick with a one-liner